src/transform/call_expr_transform.rs

```diff
@@ -118,6 +118,10 @@ impl CallExprTransform {
                     }
                     _ => None,
                 },
+
+                Expr::Ident(obj) => {
+                    replace_call_expr_if_csi_method(None, &obj, call, csi_methods, ident_provider)
```

Comment:
wouldn't be cleaner a different method? something like `replace_call_expr_if_csi_method_without_callee`?

Reply:
Yes, you're right, as my first approach were similar to the previous one, I reused the method. but with the current approach it doesn't make sense. 
I'll change it

Reply:
I changed it, wdyt?